packages/alert/src/Alert.tsx

```diff
@@ -68,7 +76,7 @@ export default function Alert({
   variant = 'success',
   ...other
 }: AlertProps): React.ReactElement<AlertProps> {
-  const Icon = variant === 'success' ? CheckIcon : ExclamationIcon
+  const Icon = iconBase[variant]
   return (
     <AlertWrapper variant={variant} type={type} {...other}>
```

Comment:
Can we give this div a `role="alert"` attribute for accessibility?

https://www.w3.org/WAI/ARIA/apg/example-index/alert/alert

Reply:
Done bbc97d899b3965a99572034686ecd68d27d23314